### DISCUSSION TITLE
Can I use scrcpy --otg to control my own phone?

### DISCUSSION THREAD

**Discussion by RebornOfc** [termux/termux-app#3772] [Q&A]
Installing scrcpy in termux, is it possible to make the command scrcpy --otg work? I wonder if I can do something like this:

1. Cast a vncserver on termux (I can do either inside or outside a proot-distro)
2. Open that vncserver at a device that has a usb mouse and keyboard connected.

Would it be possible to run scrcpy --otg to activate otg mode and control the host phone (the one with termux) using the device that is connected to the vncserver? (And my phone is not rooted if you are wondering)

**Comment by twaik** [comment]
No.
